### ISSUE TITLE
Fullscreen not saving

### ISSUE THREAD

**Issue by Cavermendes** [yuzu-emu/yuzu#5963]
New versions of Yuzu don't keep the F11 fullscreen setting after quitting. I have to manually fullscreen games everytime I exit Yuzu even after Alt-F4. Not a big deal to most, but for people running the emulator off Steam Big Picture or Launchbox/Hyperspin it's a pain.

**Comment by Its-Rei** [comment]
that is what the commandline flags are for. Just run it like:

yuzu.exe -g rom.nsp -f

**Comment by Cavermendes** [comment]
Do this through LaunchBox? I tried adding that to the command line parameters and Yuzu opens and gives an error:  "The NSP title is missing a program type NCA."

**Comment by G-Spawn** [comment]
yuzu.exe -f -g "D"\supply\game\path\example.nsp" try like that within quotes perhaps and replacing applicable info obviously. Works for playnite - other launchers.

**Comment by Morph1984** [comment]
The quotes help whenever there's a whitespace, since programs will interpret a whitespace as a delimiter when parsing command line arguments

**Comment by ZombiJambi** [comment]
I've been testing this with Steam shortcuts.
When I use 
"Path\to\yuzu.exe" -f -g "Path\to\game.nsp" OR "Path\to\yuzu.exe" -g "Path\to\game.nsp" -f: 

Yuzu window opens in a default position, then the game launches in a Desktop-resolution window, whose top left corner matches with the Yuzu window top-left corner, and forces the right-most edges of the render window off screen. Also, the taskbar is not hidden. I can correct this by pressing F11 twice.

I've tried some compatibility options on the Yuzu.exe, like 'Disable Fullscreen Optimization' and the scaling options, but nothing works.

It's as if with the command line '-f' option, it only changes the render window size, and doesn't actually do all other other fullscreen things, like center it to the screen, and hide the taskbar.

**Comment by Cavermendes** [comment]
It worked :) I disabled the option in Launchbox (No space before Rom), which makes it so -g sits right next to the rom path instead of having a space before it. Thanks for the help guys. And ZombiJambi, I guess Steam Big Picture can't handle the emulator the right way then? Because with Launchbox/BigBox I don't get any of those issues and it boots up true fullscreen with no taskbar and no windowed mode; ready to be played as normal. It's weird that this was even an issue though, is it some kind of regression with the new updates? Yuzu used to save the fullscreen setting before and I only started having this problem recently.
